openedx/openedx-wordpress-ecommerce
# docs: add the first ADRs

## Description

This PR adds 3 ADRs:
- Purpose of this Repo
- API connection
- Fulfillment and refund

And I slightly changed the index to clarify the supporting info.

I made those changes following the [ADR template from the OEP-19: Developer Documentation](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#adrs).

## Testing instructions

- Principally, I want you to help me review the content of the documents I edited.
- To see the documentation locally in the browser, you must create a python venv, install the docs.txt requirements, run `make html` inside the docs directory, and then run `python -m http.server 8200 --directory build/html`.

## Additional information
Screenshot of the index view
![Screenshot from 2023-10-24 21-28-45](https://github.com/openedx/openedx-wordpress-ecommerce/assets/35668326/92453c59-f124-4082-b45a-7e16d9b338ba)

Screenshot of the Fulfillment and refund ADR
![Screenshot from 2023-10-24 21-41-56](https://github.com/openedx/openedx-wordpress-ecommerce/assets/35668326/c86967c3-75e3-4e3d-8007-3b5bbcbbefb8)
